application/lib/registration.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	pb "github.com/refraction-networking/gotapdance/protobuf"
+	lt "../liveness"
```

Comment:
Golang doesn't really do relative path imports. This should be changed to reference the full path:
`lt "github.com/refraction-networking/gotapdance/application/liveness"`